projects/core/src/lib/components/edit/edit-dialog/edit-dialog.component.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  selectEditDialogCollapsed, selectEditDialogVisible, selectEditMapCoordinates, selectLoadingEditFeatures, selectSelectedEditFeature,
+} from '../state/edit.selectors';
+import { concatMap, filter, Observable, of, switchMap, take } from 'rxjs';
+import { expandCollapseEditDialog, hideEditDialog, updateEditFeature } from '../state/edit.actions';
+import { AppLayerModel, FeatureModel, LayerDetailsModel } from '@tailormap-viewer/api';
+import { ApplicationLayerService } from '../../../map/services/application-layer.service';
+import { FeatureWithMetadataModel } from '../models/feature-with-metadata.model';
+import { EditFeatureService } from '../edit-feature.service';
+import { selectViewerId } from '../../../state/core.selectors';
+
+@Component({
+  selector: 'tm-edit-dialog',
+  templateUrl: './edit-dialog.component.html',
+  styleUrls: ['./edit-dialog.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class EditDialogComponent implements OnInit {
+
+  public dialogOpen$: Observable<boolean> = of(false);
+  public dialogCollapsed$: Observable<boolean> = of(false);
+
+  public currentFeature$: Observable<FeatureWithMetadataModel | null> | undefined;
+  public panelWidthMargin = CssHelper.getCssVariableValueNumeric('--menubar-width') + (CssHelper.getCssVariableValueNumeric('--body-margin') * 2);
+  public layerDetails$: Observable<{ layer: AppLayerModel; details: LayerDetailsModel }> | undefined;
+
+  public loadingEditFeatureInfo$ = this.store$.select(selectLoadingEditFeatures);
+  public editCoordinates$ = this.store$.select(selectEditMapCoordinates);
+
+  public updatedFeature: FeatureModel | null = null;
+
+  constructor(
+    private store$: Store,
+    private applicationLayerService: ApplicationLayerService,
+    private editFeatureService: EditFeatureService,
+  ) {}
+
+  public ngOnInit(): void {
+    this.dialogOpen$ = this.store$.select(selectEditDialogVisible);
+    this.dialogCollapsed$ = this.store$.select(selectEditDialogCollapsed);
+    this.currentFeature$ = this.store$.select(selectSelectedEditFeature);
+    this.layerDetails$ = this.currentFeature$
+      .pipe(
+        filter((feature): feature is FeatureWithMetadataModel => !!feature),
+        switchMap(feature => {
+          return this.applicationLayerService.getLayerDetails$(feature.feature.layerId);
+        }),
+      );
+  }
+
+  public closeDialog() {
+    this.store$.dispatch(hideEditDialog());
+  }
+
+  public expandCollapseDialog() {
+    this.store$.dispatch(expandCollapseEditDialog());
+  }
+
+  public save(layerId: string) {
+    const updatedFeature = this.updatedFeature;
+    if (!updatedFeature) {
+      return;
+    }
+    this.store$.select(selectViewerId)
+      .pipe(
+        take(1),
+        concatMap(viewerId => {
+          if (!viewerId) {
+            return of(null);
+          }
+          return this.editFeatureService.updateFeature$(viewerId, layerId, updatedFeature);
```

Comment:
since we are doing a `patch` the `updatedFeature` should be  a "slim" version with only the modified attributes, not the unchanged attributes. `feature.geometry` should be omitted. 

currently the payload is eg:
```json
{
  "__fid": "begroeidterreindeel.b1e72bc0867f4c0cdfd815a7fcd5fc82",
  "geometry": "POLYGON ((131406.167 458746.464, 131417.589 458735.49, 131439.986 458758.782, 131428.495 458769.349, 131406.167 458746.464))",
  "attributes": {
    "creationdate": "2018-01-12",
    "terminationdate": null,
    "plus_status": "geenWaarde",
    "bronhouder": "G0344",
    "begroeidterreindeeloptalud": true,
    "identificatie": "G0344.85b985b90e8743cbb9f2010465fc3731",
    "geom": "POLYGON ((131406.167 458746.464, 131417.589 458735.49, 131439.986 458758.782, 131428.495 458769.349, 131406.167 458746.464))",
    "geom_kruinlijn": null,
    "bgt_status": "bestaand",
    "plus_fysiekvoorkomen": "gras- en kruidachtigen",
    "lv_publicatiedatum": "2018-01-16T21:04:56.000+00:00",
    "relatievehoogteligging": 0,
    "eindregistratie": null,
    "inonderzoek": true,
    "tijdstipregistratie": "2018-01-12T14:37:08.000+00:00",
    "class": "groenvoorziening",
    "gmlid": "b1e72bc0867f4c0cdfd815a7fcd5fc82"
  },
  "layerId": "lyr:snapshot-geoserver:postgis:begroeidterreindeel"
}
```
where only `inonderzoek` was changed, also not the layerId in there which is not part of a Feature